test/test-fdleak.c

```diff
@@ -209,6 +212,10 @@ start_client(struct event_base *base)
 {
 	struct bufferevent *bev = bufferevent_socket_new(base, -1,
                                                          BEV_OPT_CLOSE_ON_FREE);
+	if (bev == NULL) {
```

Comment:
I wish that the `main` would have a normal error path, i.e.:

```
if (!start_client)
    goto cleanup;
...
cleanup:
    if (listener)
        evconnlistener_free(listener);
    if (base)
        event_base_free(base);
```

Reply:
OK, thanks for the example